test/integration/coronavirus_local_restrictions_test.rb

```diff
@@ -181,6 +189,7 @@ def then_i_enter_a_valid_english_postcode_with_a_future_level_two_restriction
     postcode = "E1 8QS"
     stub_local_restriction(postcode: postcode,
                            name: @area,
+                           current_alert_level: 1,
```

Comment:
Ah! This is interesting! Yes, we're assuming that every local authority has an entry in the yaml now, so they should all have a current restriction.